app/cafe.py

```diff
@@ -0,0 +1,18 @@
+import datetime
+
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError, NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("NotVaccinatedError")
+        if visitor["vaccine"]["expiration_date"] < datetime.date.today():

```

Comment:
Use method `get` to avoid `KeyError`
```suggestion
        if visitor["vaccine"].get("expiration_date") < datetime.date.today():

```

Reply:
Done